tests/test_qpu.py

```diff
@@ -17,16 +17,11 @@
 
 log = logging.getLogger(__name__)
 
-pattern = "Aspen-.-[1-5]Q-."
+pattern = "Aspen-."
 VALID_QPU_LATTICES = [
     qc for qc in pyquil.list_quantum_computers() if "qvm" not in qc and re.match(pattern, qc)
 ]
 
-pattern_four_qubit = "Aspen-.-4Q-."
-VALID_FOUR_QUBIT_LATTICES = [
-    qc for qc in pyquil.list_quantum_computers() if "qvm" not in qc and re.match(pattern_four_qubit, qc)
-]
-
```

Comment:
I'm having some trouble with the tests:
- getting connection errors when running on `Aspen8`
- things are also a bit slow even on `Aspen4`, maybe because we are loading more qubits than we need to (e.g., maybe we need just 4). One suggestion is to just have:
`VALID_QPU_LATTICES = ["4q-qvm"]`?
This might speed things up and make the tests less dependent on device naming.

Reply:
I think this is a good approach, especially as it removes any server-side dependence on the test suite.

Reply:
Ok will make this change, hopefully then the tests will pass!